CI/travis.linux.after_success.sh

```diff
@@ -27,7 +27,7 @@ if [ "${DEPLOY}" = "deploy" ]; then
     exit
   fi
 
-  git clone https://github.com/Mudlet/installers.git "${TRAVIS_BUILD_DIR}/../installers"
+  git clone https://github.com/Edru2/installers.git -b patch-1 "${TRAVIS_BUILD_DIR}/../installers"
```

Comment:
Don't we want to link Mudlet git in release?

Reply:
Yes! This is just to test https://github.com/Mudlet/installers/pull/76.

Reply:
@Edru2 need to revert this now.

Reply:
Just be aware that Linux builds won't have the translation files till https://github.com/Mudlet/installers/pull/76 is not merged.

Reply:
Merged now.